src/App/store/AppState.ts

```diff
@@ -234,11 +234,11 @@ class ProjectList {
   items: Project[] = [];
 
   @observable
-  activeProjectName = '';
+  activeProjectPath = '';
```

Comment:
Is this state preserved between runs of BK? If it is then we will need to migrate the name change. I'm guessing its probably not preserved.

Reply:
No, this state does not persist. The ones that do have the `@persist` decorator.